jest.config.js

```diff
@@ -22,11 +22,15 @@ const config = {
 	moduleNameMapper: {
 		'^@/(.*)$': '<rootDir>/src/$1',
 	},
+	collectCoverage: true,
+	coverageReporters: ['text'],
```

Comment:
I would prefer `coverageReporters: ['text-summary']` to not pollute the console logs too much. What do you think?

Reply:
how about using `text-summary` as the default, but if env variable `COVERAGE_REPORT` is set to `true`, we use `text` ?

Reply:
Yeah that sounds good to me 👍Easy enough